ndk-build/CHANGELOG.md

```diff
@@ -36,7 +37,7 @@
 
 - **Breaking:** Default `target_sdk_version` to `30` or lower (instead of the highest supported SDK version by the detected NDK toolchain)
   for more consistent interaction with Android backwards compatibility handling and its increasingly strict usage rules:
-  <https://developer.android.com/distribute/best-practices/develop/target-sdk>
+  https://developer.android.com/distribute/best-practices/develop/target-sdk
```

Comment:
We've already committed to these now, fine to leave them.

(As mentioned elsewhere in this / the other PR, vscode stops highlighting them in the code view without these, so it seems editors are expecting these markers too)